binamb.planx-pla.net/manifests/hatchery/hatchery.json

```
@@ -0,0 +1,65 @@
+{
+    "user-namespace": "jupyter-pods-binamb",
+    "sub-dir": "/lw-workspace",
+    "user-volume-size": "10Gi",
+    "sidecar": {
+        "cpu-limit": "1.0",
+        "memory-limit": "256Mi",
+        "image": "quay.io/cdis/ecs-ws-sidecar:master",
+        "env": {
+            "NAMESPACE": "binamb",
+            "GEN3_ENDPOINT": "binamb.planx-pla.net",
+            "HOSTNAME": "binamb.planx-pla.net"
+        },
+        "args": [],
+        "command": [
+            "/bin/bash",
+            "/sidecar.sh"
+        ],
+        "lifecycle-pre-stop": [
+            "su",
+            "-c",
+            "echo test",
+            "-s",
+            "/bin/sh",
+            "root"
+        ]
+    },
+    "containers": [
+        {
+            "target-port": 8888,
+            "cpu-limit": "1.0",
+            "memory-limit": "2Gi",
+            "name": "(Generic) Jupyter Lab Notebook with R Kernel",
+            "image": "quay.io/cdis/jupyter-superslim-r:master",
+            "env": {
+                "FRAME_ANCESTORS": "https://binamb.planx-pla.net"
+            },
+            "args": [
+                "--NotebookApp.base_url=/lw-workspace/proxy/",
+                "--NotebookApp.default_url=/lab",
+                "--NotebookApp.password=''",
+                "--NotebookApp.token=''",
+                "--NotebookApp.ip='*'",
+                "--NotebookApp.notebook_dir='/home/jovyan'",
+                "--NotebookApp.shutdown_no_activity_timeout=5400",
+                "--NotebookApp.quit_button=False"
```

```suggestion
                "--JupyterNotebookApp.quit_button=False"
```
I think we should substitute all those `NotebookApp` with `JupyterNotebookApp` for the updated jupyter notebook image